contracts/hardhat.config.js

```diff
@@ -87,10 +87,9 @@ module.exports = {
       },
       { // for polyjuice contracts
         version: "0.6.6",
-        settings: {}
       },
       { version: "0.7.5" },
       { version: "0.8.6" }
-    ], overrides: {}
+    ]
```

Comment:
missing to code about 10 minutes？

Reply:
The setting of timeout never works and I don't know why... the timeout is `40000ms` the defailt timeout used by hardhat.